website/16s-rrna-amplification-protocol.md

```diff
@@ -0,0 +1,201 @@
+#### Version 4\_13
+
+Primers for paired-end 16s community sequencing on the Illumina HiSeq
+platform using bacteria/archaeal primers 515F/806R. Please see this
+article: [Caporaso JG, Lauber CL, Walters WA, Berg-Lyons D, Huntley J,
+Fierer N, Owens SM, Betley J, Fraser L, Bauer M, Gormley N, Gilbert JA,
+Smith G, Knight R. 2012. Ultra-high-throughput microbial community
+analysis on the Illumina HiSeq and MiSeq platforms. **ISME
+J.**](http://www.nature.com/ismej/journal/vaop/ncurrent/full/ismej20128a.html)
+For running these libraries on the MiSeq and HiSeq please make sure you
+read the supplementary methods of the above manuscript very well – you
+will need to make your sample more complex by adding **5-10%** PhiX to
+your run.
+
+Download Links
+--------------
+
+-   [EMP DNA Extraction
+    Protocol](http://press.igsb.anl.gov/earthmicrobiome/files/2013/04/EMP_DNA_extraction_protocol_version_4_13.doc)
+-   [New 16S Illumina Amplification
+    Protocol](https://dl.dropboxusercontent.com/u/68839641/emp_website/515f_806_16S_illumina_amplification_protocol_version_6_15.doc)
+-   [Old 16S Illumina Amplification
+    Protocol](http://press.igsb.anl.gov/earthmicrobiome/files/2013/04/515f_806_16S_illumina_amplification_protocol_version_4_13.doc)
+-   [Old Illumina HiSeq 16S Primer
+    Sequences](ftp://ftp.metagenomics.anl.gov/data/misc/EMP/SupplementaryFile1_barcoded_primers_515F_806R.txt)
+    (515f - 806r)
+-   [New Illumina HiSeq 16S Primer
+    Sequences](https://dl.dropboxusercontent.com/u/68839641/emp_website/515f_806rb_new.xls)
+    (515f - 806rB)
+-   [New Illumina HiSeq 16S Primer
+    Sequences](https://dl.dropboxusercontent.com/u/68839641/emp_website/515f_926r_new.xlsx)
+    (515f - 926r)
+-   [Primer Ordering and
+    Resuspension](http://press.igsb.anl.gov/earthmicrobiome/files/2013/04/EMP_primer_ordering_and_resuspension.doc)
+
+Primer Constructs designed by Greg Caporaso
+-------------------------------------------
```

Comment:
I agree. Let's place all the current (new) primers in the `.md` file. For the original (old) primers, we could place them at the bottom of the page or leave them as links.